webapi/tct-backgrounds-css3-tests/backgrounds/csswg/reference/background-size-009-ref.html

```diff
@@ -1,15 +0,0 @@
-<!DOCTYPE html>
```

Comment:
There is a same file at misc/webapi-uiautomation-xwalk-tests/css3-background/reference/background-size-009-ref.html, which referenced by
- misc/webapi-uiautomation-xwalk-tests/tests.xml
- misc/webapi-uiautomation-xwalk-tests/css3-background/csswg/background-size-009.html
- misc/webapi-uiautomation-xwalk-tests/css3-background/csswg/background-size-017.html
- misc/webapi-uiautomation-xwalk-tests/css3-background/csswg/background-size-014.html

But this file isn't referenced by tct-backgrounds-css3-tests